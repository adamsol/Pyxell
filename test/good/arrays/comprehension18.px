
print [for a in [[]] yield ...a]

a = [false, true]
print [for i in 0..2 yield ...a*i*i by -i]

print [for i in 0...5 for j in 0...4 if j != 1 yield ..."qwerty"].length
