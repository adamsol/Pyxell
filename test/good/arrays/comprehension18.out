[]
[true, false, true, true, true, true]
90
